content/blog/01_mapmaking.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Map Making"
+date = 2021-05-01
+
+extra.image = "assets/images/blog/mapmaking.png"
++++
+
+Map Making 101
+
+When a builder decides that they want to make a map, they first of all, head to our (whitelisted) build server found at <code>build.nucleoid.xyz</code>
+
+They choose a game and create a map using a handy command.
+
+<br>
+<img class="header_image" src="https://iili.io/Kg34kl.jpg">
+
+One of our builders is creating a [KotH (King of the Hill)](https://github.com/NucleoidMC/koth) map using [Plasmid](https://github.com/NucleoidMC/plasmid). Plasmid is our custom game library which allows us to create minigames, and maps for those minigames, with ease.
+
+Plasmid allows our builders to access a <code>/map</code> command. The map command allows the swift creation of maps, editing, and exporting for use in production. 
+
+Each map requires regions (Unless the game doesn't require any set regions) which can be configured using the <code>/map region</code> command.
```

Comment:
I think it might be nice to describe what a region means and the importance of it- the details of what the command is specifically is not super important here to the casual reader I imagine.

Reply:
This blog gets on the technical side, i plan to make a blog on how the server builds the mods on restart/start which wont be for the faintest of readers.

Reply:
Yeah- though the specific of the name of the command is not so interesting technically as opposed to the concept of what a region means.